packages/web-service/src/routes/routes.js

```diff
@@ -19,6 +20,7 @@ const routes = [
   ...notEligibleProject,
   ...eligibilityCheck,
   ...eligible,
+  ...tasklist,
```

Comment:
I can see this list starting to get pretty long and potentially not telling us a great deal - is it worth looking at doing something like file scanning for routes, like https://www.npmjs.com/package/hapi-auto-route ?